packages/fuels-types/src/lib.rs

```diff
@@ -17,6 +17,8 @@ pub mod utils;
 // Since it would have the format `struct foo` or `enum bar`, there is a whitespace.
 pub const STRUCT_KEYWORD: &str = "struct ";
```

Comment:
Although these were once used elsewhere, none of the four now have a usage outside the one place inside this file. 

I'd inline these constants to their only place of usage and, in doing so, clarify their meaning and make the code more readable.